coil-base/src/main/java/coil/memory/ViewTargetRequestManager.kt

```diff
@@ -67,7 +67,7 @@ internal class ViewTargetRequestManager : View.OnAttachStateChangeListener {
         currentRequestJob = null
 
         pendingClear?.cancel()
-        pendingClear = CoroutineScope(Dispatchers.Main.immediate).launch { setCurrentRequest(null) }
+        pendingClear = GlobalScope.launch(Dispatchers.Main.immediate) { setCurrentRequest(null) }
```

Comment:
Using `GobalScope` isn't ideal, but it's basically equivalent to `CoroutineScope(Dispatchers.Main.immediate).launch` so might as well save an allocation.